egui_demo_lib/src/apps/demo/misc_demo_window.rs

```diff
@@ -1,12 +1,17 @@
 use super::*;
-use egui::{color::*, *};
+use crate::LOREM_IPSUM;
+use egui::{color::*, epaint::text::TextWrapping, *};
 
 /// Showcase some ui code
 #[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
 #[cfg_attr(feature = "serde", serde(default))]
 pub struct MiscDemoWindow {
     num_columns: usize,
 
+    break_anywhere: bool,
+    max_lines: usize,
+    overflow_character: String,
```

Comment:
```suggestion
    overflow_character: char,
```

will make the rest of the code simpler